processor/transformprocessor/internal/common/logs.go

```diff
@@ -105,7 +105,11 @@ func (pc LogParserCollection) ParseContextStatements(contextStatements ContextSt
 		if err != nil {
 			return nil, err
 		}
-		lStatements := ottllog.NewStatementSequence(parsedStatements, pc.settings, ottllog.WithStatementSequenceErrorMode(pc.errorMode))
+		parsedGlobalConditions, err := pc.logParser.ParseConditions(contextStatements.Conditions)
```

Comment:
We shouldnt need to expose any new public functions in OTTL.  You can use the functions in `internal/filter/filterottl` to execute a boolean condition statement(s).

Reply:
Thanks for your suggestion, I was able to get the same effect without adding new public functions in OTTL.